sbt-plugin/src/main/scala/org/scalajs/sbtplugin/ScalajspUtils.scala

```diff
@@ -1,66 +0,0 @@
-/*
- * Scala.js (https://www.scala-js.org/)
- *
- * Copyright EPFL.
- *
- * Licensed under Apache License 2.0
- * (https://www.apache.org/licenses/LICENSE-2.0).
- *
- * See the NOTICE file distributed with this work for
- * additional information regarding copyright ownership.
- */
-
-package org.scalajs.sbtplugin
-
-import scala.collection.mutable
-
-import sbt._
-import sbt.complete._
-
-import org.scalajs.io._
-
-private[sbtplugin] object ScalajspUtils {
-
-  /** Creates an [[ExampleSource]] with per-directory tab completion. */
-  def relPathsExamples(relPaths: Seq[String]): ExampleSource =
-    new ScalaJSIRFilesOnClasspathExamples(relPaths)
-
-  /** An [[ExampleSource]] showing .sjsir files on a classpath. */
-  private class ScalaJSIRFilesOnClasspathExamples(allRelPaths: Seq[String],
-      prefix: String = "") extends ExampleSource {
-
-    override def apply(): Iterable[String] = {
-      val allExamples = (for {
-        relPath <- allRelPaths
-        if relPath.startsWith(prefix)
-      } yield {
-        // Returned examples must not include the prefix
-        val remaining = relPath.substring(prefix.length)
-
-        /* For files in subdirectories wrt. the current prefix, do not show
-         * the entire remaining path. Instead, show only the remaining path
-         * to the subdirectory, '/' included. This means that:
-         *
-         * > scalajsp hello<tab>
-         *
-         * will complete to
-         *
-         * > scalajsp helloworld/
-         *
-         * and further tabs are necessary to show to files and directories
-         * under the helloworld/ subdirectory.
```

Comment:
This behavior would still be welcome with decoded names, with separations based on `.` instead of `/`.

Reply:
Well yes, but is it worth the additional code and divergence from standard sbt behavior? (e.g. `testOnly` does not do that).

Reply:
Ah, I was under the impression that it did, but you're right, it doesn't do that.